ros2service/ros2service/verb/find.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.node.strategy import NodeStrategy
+from ros2srv.api import service_type_completer
```

Comment:
This import needs to move two lines down to be in alphabetical order so that CI passes: https://ci.ros2.org/job/ci_linux/7325/testReport/junit/ros2service.test/test_flake8/test_flake8/

Reply:
Done.